src/Control/Carrier/Parse/Measured.hs

```diff
@@ -69,7 +68,6 @@ runParser blob@Blob{..} parser = case parser of
         >>= either (\e -> trace (displayException e) *> throwError (SomeException e)) pure
 
   AssignmentParser    parser assignment -> runAssignment Assignment.assign    parser blob assignment
-  DeterministicParser parser assignment -> runAssignment Deterministic.assign parser blob assignment
```

Comment:
Since deterministic assignment is gone, so is the corresponding `Parser` constructor 🔥